conda/install.py

```diff
@@ -922,6 +927,8 @@ def move_path_to_trash(path):
         except OSError as e:
             log.debug("Could not move %s to %s (%s)" % (path, trash_dir, e))
         else:
+            if path in linked_data_:
+                del linked_data_[path]
```

Comment:
`path` can be anything, but it only deletes it when it's found as a key in `linked_data_` which will correspond to `path` being a `prefix`.

I'll take a look at the `conda-meta` directory detail now.